### PR TITLE
fixed according to Prospector

### PR DESCRIPTION
**Pull request by VladyslavHnatchenko**
Hello @robvanderleek!

Thank you very much, for your article(https://medium.com/bettercode/how-to-build-a-modern-ci-cd-pipeline-5faa01891a5b)!
I corrected the files a bit according to Prospector tool (https://prospector.readthedocs.io/en/master/#). 

Please check.
Thank you.

### CODE REVIEW

File: tests/test_generator.py [hunk 1]
```diff
@@ -1,20 +1,26 @@
...
 import unittest
-
 from buzz import generator
 
+
 def test_sample_single_word():
-    l = ('foo', 'bar', 'foobar')
-    word = generator.sample(l)
-    assert word in l
+    """Test simple word."""
```
Comment (robvanderleek) [inline tests/test_generator.py]: Please remove the comment (and optionally rename the test function)

File: tests/test_generator.py [hunk 2]
```diff
@@ -1,20 +1,26 @@
...
+    list_ = ('foo', 'bar', 'foobar')
+    word = generator.sample(list_)
+    assert word in list_
+
 
 def test_sample_multiple_words():
-    l = ('foo', 'bar', 'foobar')
-    words = generator.sample(l, 2)
+    """Test multiple words."""
```
Comment (robvanderleek) [inline tests/test_generator.py]: Please remove the comment (and optionally rename the test function)

File: tests/test_generator.py [hunk 3]
```diff
@@ -1,20 +1,26 @@
...
     assert len(words) == 2
-    assert words[0] in l
-    assert words[1] in l
+    assert words[0] in list_
+    assert words[1] in list_
     assert words[0] is not words[1]
 
+
 def test_generate_buzz_of_at_least_five_words():
+    """Test phrase with at least 5 words."""
```
Comment (robvanderleek) [inline tests/test_generator.py]: Please remove the comment (and optionally rename the test function)

**Review by robvanderleek — CHANGES_REQUESTED** [review]
Hi Vladyslav,

Thanks for your PR! 
I try to keep the names of unit-test functions as descriptive as possible, so I don't need comments explaining their intent.

Can you have a look? Apart from that the other changes are nice improvements!
Rob

**Comment by VladyslavHnatchenko** [comment]
Hi Rob,
I deleted comments.
Thanks and have a nice day!

**Review by robvanderleek — APPROVED** [review]

**Pull request merged**